README.md

```diff
@@ -60,21 +101,28 @@ $ git clone https://TDEI-UW@dev.azure.com/TDEI-UW/TDEI/_git/file-service
 
 ## Secrets
 
-Application secrets are not included in the code repository. Below are the instruction for each environment
+Application secrets are not included in the code repository. File service relies on below-mentioned environment variable
 
-###### Local
+| Environment Variable                       | Description                                        |
+|--------------------------------------------|----------------------------------------------------|
+| GTFS_FLEX_CONTAINER_NAME                   | Name of the blob container to upload file          |
+| GTFS_FLEX_UPLOAD_ALLOWED_EXTENSIONS        | File extension allowed for upload. comma separated |
+| GTFS_FLEX_UPLOAD_TOPIC_NAME                | Name of the upload topic to publish message        |
+| GTFS_PATHWAYS_CONTAINER_NAME               | Name of the blob container to upload file          |
+| GTFS_PATHWAYS_UPLOAD_ALLOWED_EXTENSIONS    | File extension allowed for upload. comma separated |
+| GTFS_PATHWAYS_UPLOAD_TOPIC_NAME            | Name of the upload topic to publish message        |
+| OSW_CONTAINER_NAME                         | Name of the blob container to upload file          |
+| OSW_UPLOAD_ALLOWED_EXTENSIONS              | File extension allowed for upload. comma separated |
+| OSW_UPLOAD_TOPIC_NAME                      | Name of the upload topic to publish message        |
+| CLOUD_AZURE_STORAGE_BLOB_CONNECTION_STRING | Azure storage connection string                    |
+| CLOUD_AZURE_SERVICE_BUS_CONNECTION_STRING  | Azure service bus connection string                |
 
```

Comment:
How to get sample values for these environment variables? or How to set locally?